### PR TITLE
Add Krytocon

### PR DESCRIPTION
**Pull request by ausmacon**

### CODE REVIEW

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -344,6 +344,22 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+			for (const target of pokemon.adjacentFoes()) {
+				if (!activated) {
+					this.add('-ability', pokemon, 'Curse of Dexit');
+					activated = true;
+				}
+				target.addVolatile('curse');
+			}
+		},
+	},
+
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts:362]:
```suggestion

```

File: data/mods/ssb/abilities.ts [hunk 2]
```diff
@@ -344,6 +344,22 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+	curseofdexit: {
+		name: "Curse of Dexit",
+		shortDesc: "User sets Curse against foe on entry; 25% of max HP lost.",
+		onStart(pokemon) {
+			this.directDamage(pokemon.maxhp / 4, pokemon, pokemon);
+			for (const target of pokemon.adjacentFoes()) {
+				if (!activated) {
+					this.add('-ability', pokemon, 'Curse of Dexit');
+					activated = true;
+				}
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts:357]: this seems unintentional? activated is never defined

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -516,6 +516,80 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		target: "self",
 		type: "Fairy",
 	},
+
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts:519]:
```suggestion

```

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -516,6 +516,80 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+								break;
+							}
+						}
+					}
+					this.actions.runMove('attackofopportunity', source, source.getLocOf(pokemon));
+				}
+			},
+		},
+		onAfterMoveSecondarySelf(pokemon, target, move) {
+			if (!target || target.fainted || target.hp <= 0 || target.beingCalledBack || target.switchFlag) this.boost({atk: 2}, pokemon, pokemon, move);
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts:586]:
```suggestion
			if (!target || target.fainted || target.hp <= 0 ||
				target.beingCalledBack || target.switchFlag) {
				this.boost({atk: 2}, pokemon, pokemon, move);
			}
```

File: data/mods/ssb/pokedex.ts [hunk 5]
```diff
@@ -49,6 +49,12 @@ export const Pokedex: {[k: string]: ModdedSpeciesData} = {
 		baseStats: {hp: 110, atk: 200, def: 99, spa: 101, spd: 99, spe: 35},
 		abilities: {0: "Cacophony"},
 	},
+
```
Comment (KrisXV) [inline data/mods/ssb/pokedex.ts:52]:
```suggestion

```

File: data/mods/ssb/abilities.ts (hunk 2 above)
Comment (ausmacon) [reply data/mods/ssb/abilities.ts:357]: unintended yea mb

File: data/mods/ssb/moves.ts [hunk 6]
```diff
@@ -516,6 +516,82 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		target: "self",
 		type: "Fairy",
 	},
+	// Krytocon
```
Comment (HisuianZoroark) [inline data/mods/ssb/moves.ts:519]: make sure you add to the pursuit clones list in scripts as well

**Pull request merged**